snippets/other.cson

```diff
@@ -0,0 +1,108 @@
+'.source.coffee':
```

Comment:
Generally this file needs some work. There are spelling mistakes and I don't agree on having shorthand only